app/controllers/clients_controller.rb

```diff
@@ -13,6 +13,7 @@ def index
   def new
     @client = Client.new
     @client.address = Address.new
+    @client.contact_details << ContactDetail.new
```

Comment:
Rails way of doing `@client.contact_details  << ContactDetail.new` is just `@client.contact_details.build`